src/main/resources/META-INF/plugin.xml

```diff
@@ -216,6 +216,22 @@
             <!-- In $default keymap `control` key (Windows/Linux) corresponds to `cmd` key on macOS -->
             <keyboard-shortcut keymap="$default" first-keystroke="control alt shift M" />
         </action>
+
+        <action id="GitMachete.CheckoutNextAction"
+                class="com.virtuslab.gitmachete.frontend.actions.navigation.CheckoutNextAction">
+            <keyboard-shortcut keymap="$default" first-keystroke="alt DOWN" />
+            <keyboard-shortcut keymap="$default" first-keystroke="alt RIGHT" />
```

Comment:
@vzmerr I hope you still have patience for this one 😆 

as discussed on slack (take a look to the channel if you have not seen it yet)...

Let's be consistent with git machete CLI - if I understood correctly it means that the right arrow should go to the next sibling (not to the child)

Reply:
@PawelLipski suggests right arrow to go to the first child

Reply:
+1, right arrow to first child if such child exists, otherwise do nothing